generate-errors/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "generate-errors"
 authors = [
-  "Bevy Contributors <bevyengine@gmail.com>",
+  "Bevy Contributors <support@bevyengine.org>",
   "Carter Anderson <mcanders1@gmail.com>",
```

Comment:
```suggestion
  "Carter Anderson <cart@bevyengine.org>",
```

Reply:
I'm actually still using my other email for github stuff.